requirements-dev.txt

```diff
@@ -15,7 +15,8 @@ stestr>=2.0.0,!=4.0.0
 pylatexenc>=1.4
 ddt>=1.2.0,!=1.4.0,!=1.4.3
 seaborn>=0.9.0
-reno>=3.4.0
+# TODO: switch to stable release when 4.1 is released
+reno @ git+https://github.com/openstack/reno.git@81587f616f17904336cdc431e25c42b46cd75b8f
```

Comment:
This is fine to use the github mirror if the commit is there (TBH I thought they stopped mirroring the repos to github). Was there a reason you didn't use https://opendev.org/openstack/reno as the git source?

Reply:
They still update it: https://github.com/openstack/reno

>  Was there a reason you didn't use https://opendev.org/openstack/reno as the git source?

Yes, laziness. I've used VCS requirements many times with GitHub and know exactly how to do it. I was lazy in not wanting to learn how to do it in Opendev—given that it took me a while to get set up the first time with Reno—even though this is probably extremely similar